.gitignore

```diff
@@ -14,8 +14,12 @@ docs/crowdin.py
 *.jpg
 *.flac
 *.mo
-pyproject.toml
+.env
+*.ts
+tsconfig.json
 dist/
+pyproject.toml
+*mp4
```

Comment:
like what are mp4's doing here

Reply:
@ooliver1 I remember there was a conversation to support mp4 or not in voice rec just added is Incase

Reply:
Generally you don't find people storing the mp4's in the same dir as the repo, anyway if you're bothering to do that there's a lot more formats

Reply:
Oh also keep PR's to one issue/topic maybe

Reply:
Ah Okay